codec-http/src/main/java/io/netty/handler/codec/http/cookie/ClientCookieEncoder.java

```diff
@@ -34,9 +34,6 @@
  * A <a href="http://tools.ietf.org/html/rfc6265">RFC6265</a> compliant cookie encoder to be used client side, so
  * only name=value pairs are sent.
  *
- * User-Agents are not supposed to interpret cookies, so, if present, {@link Cookie#rawValue()} will be used.
```

Comment:
Because `Cookie#rawValue` doesn't exist. This is something that existed in some previous implementation in AHC. I think it was removed along the review when I started contributing it to Netty, the code went away but the javadoc stayed.